packages/rancher-cis-benchmark/charts/templates/benchmark-cis-1.24.yaml

```
@@ -0,0 +1,8 @@
+---
+apiVersion: cis.cattle.io/v1
+kind: ClusterScanBenchmark
+metadata:
+  name: cis-1.24
+spec:
+  clusterProvider: ""
+  minKubernetesVersion: "1.24.0"
```

```suggestion
  minKubernetesVersion: "1.24.0"
  maxKubernetesVersion: "1.24.x"
```
In preparation of 1.7, and as 1.8 is being developed (CIS Workbench) it is safe to use `maxKubernetesVersion: "1.24.x"` for cis-1.24 as well. 

See CIS Workbench 1.24 Workbench full description [here](https://workbench.cisecurity.org/benchmarks/10873).

---

Updated the PR as suggested.